features/old/accounts/invoices.feature

```diff
@@ -39,22 +39,24 @@ Feature: Show invoices from account's page (#16015909)
 
   Scenario: Display years of invoices when there are invoices
     Given a provider is logged in on 1st January 2011
-    Given a buyer "zoidberg" signed up to provider "foo.3scale.localhost"
+    And a buyer "zoidberg" signed up to provider "foo.3scale.localhost"
     And the provider is charging its buyers in prepaid mode
     And an invoice of buyer "zoidberg" for January, 2011
-    And an invoice of buyer "zoidberg" for January, 2012
-    And I go to the invoices issued by me
-    Then I should see the list of years with invoices have the following years:
-      | 0  |
+    And an invoice of buyer "zoidberg" for January, 2014 on 1st January 2014 (without jobs)
+    When I go to the invoices issued by me
+    Then the years selector should have the following options:
+      |      |
```

Comment:
The user sees an empty value, so in my opinion stating in the scenario that it is `0` is incorrect.

Reply:
yes it show empty on view but it's value is zero in its HTML, but yes you are correct it looks incorrect to show 0 . I agree gere